frontend/galeri/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
I normally use [gitignore.io](https://www.gitignore.io/).

Reply:
This was autogenerated